.github/workflows/misc-update-wishlist-leaderboard.yml

```diff
@@ -0,0 +1,36 @@
+name: Update wishlist leaderboard
+
+on:
+  schedule:
+    # Run every 3 hours
```

Comment:
I think every day is fine, but probably this is cheap af :D

Reply:
The job takes ~1 minute 15 secs to run.

Reply:
default `ubuntu-latest` are free for public repos